crates/ruff_python_parser/src/parser/expression.rs

```diff
@@ -498,9 +528,13 @@ impl<'src> Parser<'src> {
                     }
                 }
             },
+            true,
```

Comment:
This is not related to this PR and more feedback for myself. I just had to navigate to the definition to figure out what the meaning of `true` is here. I don't think it's a problem in IDEs but it's difficult to know the meaning when reading the code. Two suggestions (for a separate PR)

* Create an enum `TrailingComma` with variants `Allowed`, `Forbidden`
* We could move it into `RecoveryContextKind` because it should be the same in all places where we use the same kind.